services/one-stream/handleOneStream.js

```diff
@@ -139,14 +108,39 @@ export const main = async (eventBatch) => {
           );
           return;
         }
-        if (typeof BUILD_PACKAGE_CALL[packageToBuild.type] === "function")
-          await BUILD_PACKAGE_CALL[packageToBuild.type](packageToBuild.id);
-        else
-          console.log(
-            `%s with type <%s> has no build package function??`,
-            inPK,
-            packageToBuild.type
-          );
+        switch (packageToBuild.type) {
```

Comment:
This makes me sad that the tool forced us to break this out due to fake security concerns. :(

Reply:
Me too - I had flagged it as a "false positive" but the idea that it was in the reporting just kept nagging at me :(